cibyl/plugins/openstack/printers/serialized.py

```diff
@@ -49,16 +49,16 @@ def print_deployment(self, deployment: Deployment) -> str:
                 'security_group': network.security_group.value,
                 "dvr": network.dvr.value,
                 'tls_everywhere': network.tls_everywhere.value
-            },
+            } if network else {},
             'storage': {
                 'cinder_backend': storage.cinder_backend.value
-            },
+            } if storage else {},
             'ironic': {
                 'ironic_inspector': ironic.ironic_inspector.value,
                 'cleaning_network': ironic.cleaning_network.value
-            },
+            } if ironic else {},
             'overcloud_templates':
-                list(deployment.overcloud_templates.value)
+                [deployment.overcloud_templates.value]
```

Comment:
I think this should be reverted back to calling `list`. Since the templates are stored in a set, we need to convert it to a list before serializing, otherwise the json module crashes:
`TypeError: Object of type set is not JSON serializable`
because we pass a list of sets.

Reply:
Done.